setup.php

```diff
@@ -20,67 +20,114 @@
+								environment.</p>
+							</div>
 						</div>
 					</div>
-				</div>
-				<div class="row">
-					<div class="panel panel-default col-xs-offset-1 col-xs-10">
-						<div class="panel-heading">
-							<h3 class="panel-title">Administrator Credentials</h3>
-						</div>
-						<div class="panel-body">
-							<p><?php echo $ProjectName ?> currently has an Administrator
-							which can configure the parameters of the software. Please
-							enter the user credentials below.</p>
-							<p>Make sure the credentials you use are secure and cannot
-							be guessed by employees and/or customers who may obtain
-							access to the service.</p>
-							<div class="row">
-								<div class="form-group col-xs-offset-1 col-xs-10">
-									<label for="username">Username:</label>
-									<input type="text" class="form-control" id="username" name="username"></input>
-								</div>
-								<div class="form-group col-xs-offset-1 col-xs-10">
-									<label for="password">Password:</label>
-									<input type="password" class="form-control" id="password" name="password"></input>
-								</div>
-								<div class="form-group col-xs-offset-1 col-xs-10">
-									<label for="password2">Repeat Password:</label>
-									<input type="password2" class="form-control" id="password2" name="password2"></input>
+					<div class="row">
+						<div class="panel panel-default col-xs-offset-1 col-xs-10">
+							<div class="panel-heading">
+								<h3 class="panel-title">Administrator Credentials</h3>
+							</div>
+							<div class="panel-body">
+								<p><?php echo $ProjectName ?> currently has an Administrator
+								which can configure the parameters of the software. Please
+								enter the user credentials below.</p>
+								<p>Make sure the credentials you use are secure and cannot
+								be guessed by employees and/or customers who may obtain
+								access to the service.</p>
+								<div class="row">
+									<div class="form-group col-xs-offset-1 col-xs-10">
+										<label for="username">Username:</label>
+										<input type="text" class="form-control" id="username" name="username"></input>
+									</div>
+									<div class="form-group col-xs-offset-1 col-xs-10">
+										<label for="password">Password:</label>
+										<input type="password" class="form-control" id="password" name="password"></input>
+									</div>
+									<div class="form-group col-xs-offset-1 col-xs-10">
+										<label for="password2">Repeat Password:</label>
+										<input type="password" class="form-control" id="password2" name="password2"></input>
+									</div>
 								</div>
-
+									<button type="submit" class="btn btn-info col-xs-offset-1 center-block">Submit</button>
 							</div>
 						</div>
 					</div>
+				</form>
+			<?php }else{
+				class MyDB extends SQLite3{
+					function __construct(){
+					    $this->open($GLOBALS['DBname']);
+					}
+				}
+				$db = new MyDB();
```

Comment:
You might want to create an API of some sort that will be used internally:

Use the `database.php` file above to add functions like `initializeDatabaseTables();` and `createAdminUser();`. That way you do not need to repeat code you've already written, therefore making the project more maintainable, as well as faster and more secure to develop.